FindImportProfile/FindImportProfile.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  PluginFindRecord,
+  PluginFindRecordModal,
+} from '@folio/stripes-acq-components';
+
+import { ENTITY_KEYS } from '@folio/data-import/src/utils/constants';
```

Comment:
I think it makes sense to remove the line above and add one below as this import kind of third-party. If you have a convention for this, then it is fine.

Reply:
Done